omigami/feature_selector.py

```diff
@@ -29,11 +39,10 @@ def __init__(
         self.is_fit = False
         self.random_state = None if random_state is None else RandomState(random_state)
         self.n_outer = n_outer
-        self.metric = metric
-        self.estimator = estimator
-        self.features_dropout_rate = features_dropout_rate
+        self.keep_fraction = 1 - features_dropout_rate
         self.robust_minimum = robust_minimum
         self.repetitions = repetitions
+        self.feature_evaluator = FeatureEvaluator(estimator, metric, random_state)
 
         if not n_inner:
             logging.warning("n_inner is not specified, setting it to n_outer - 1")
```

Comment:
Warning with default parameters is not great imho maybe reduce this to debug, or make n_inner required, favouring latter.